### DISCUSSION TITLE
Could it work with Sensostar 2?

### DISCUSSION THREAD

**Discussion by stefinuxha** [Zeppelin500/MBusino#20] [Ideas]
![kompakt-waermezaehler-engelmann-sensostar-2-qn-qp-06-52-mm](https://github.com/user-attachments/assets/50f98c8d-f707-4c42-9a63-f94734a416e5)

Hi all, very nice project, do you think it could work with Sensorstar 2? My meter reads cold water (cubic m), hot water (cubic m) and calculate the kWh of the central heating system used by my apartment. I think also sensorstar 2 shall have Mbus according to its tiny text at the bottom:
<img width="544" alt="Screenshot 2024-10-16 alle 18 57 14" src="https://github.com/user-attachments/assets/4bf0cb98-0609-4bd8-905f-d9aa7ee776f8">

**Comment by Zeppelin500** [comment]
I do not know the heatmeter. Is there a open 2 wire cable? This is M-Bus.
Open the cover, look inside and share the picture.
May be the optical interface use the M-Bus protocol?
MBusino support only wired M-Bus. But, of you have a wired M-Bus, it should work.
